exercises/03-redux/src/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import { Root } from 'modules/root/components/root'
+import { AddUser } from 'modules/users/user-types'
+
+import { createStore, compose } from 'redux'
+
+import { rootReducer, RootState } from 'modules/root/root-reducer'
+
+import { UserActionCreators } from 'modules/users/user-actions'
+
+// add redux devtools to window
+declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose
+  }
+}
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
+
+const store = createStore(rootReducer, composeEnhancers())
+
+const dispatchAddUser: AddUser = (user) =>
+  store.dispatch(UserActionCreators.addUser(user))
+
+const render = (state: RootState) => {
+  const { title, users } = state.users
```

Comment:
This can be destructured directly in the arguments.

Reply:
It seemed better readable compared to 
```ts
const render = ({ users: { title, users } }: RootState) => // ...
```

Rewritten in 0b2be098dc7a5b317c6eafdc69eb9284647cea5a

Reply:
Yeah, it's just personal preference. Both are fine.